lib/pf/UnifiedApi/Controller/Nodes.pm

```diff
@@ -1051,6 +1051,11 @@ sub import_item {
         node_update_last_seen($mac);
     }
 
+    if (exists($node{-ip4})) {
```

Comment:
@jrouzierinverse should we namespace this key? `ip4` => `ip4log.ip` ?

Reply:
That makes sense.
That should be changed to ip4log.ip
```if (exists($node{'ip4log.ip'})) {```